lxd/device/disk.go

```diff
@@ -552,6 +559,60 @@ func (d *disk) createDevice() (string, error) {
 	isFile := false
 	if d.config["pool"] == "" {
 		isFile = !shared.IsDir(srcPath) && !IsBlockdev(srcPath)
```

Comment:
Do we need the isFile check here anymore? maybe it is somehow creating a file instead of a path?

Reply:
Ah right, this part is wrong, `isFile` should always be set to `false` when in either ceph or cephfs.
The logic is still correct for all other cases though so should be kept.

Reply:
Updated